msgraphforentra.json

```
@@ -0,0 +1,1008 @@
+{
+    "appid": "bf5de8fd-6437-4fc9-8c33-5f5bd3971d26",
+    "name": "MS Graph for Entra",
+    "description": "This app integrates with Microsoft Entra to execute various generic and investigative actions",
+    "type": "endpoint",
+    "product_vendor": "Microsoft",
+    "logo": "logo_msgraphforentra.svg",
+    "logo_dark": "logo_msgraphforentra_dark.svg",
+    "product_name": "Microsoft Entra",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk",
+    "contributors": [
+        {
+            "name": "Ian Wills"
+        }
+    ],
+    "license": "Copyright (c) 2022-2024 Splunk Inc.",
```

I hope to have a more proactive solution for this before 2026, but things are gonna get mad at 2024 now
```suggestion
    "license": "Copyright (c) 2022-2025 Splunk Inc.",
```